content/learn/getting-started/npm.pt.md

```diff
@@ -0,0 +1,131 @@
+---
+title: an-introduction-to-the-npm-package-manager
+displayTitle: 'Uma introdução ao gestor de pacote NPM'
+description: 'Um guia rápido ao npm, o poderoso gestor de pacote que tornou-se a chave para o sucesso da Node.js. Em Janeiro de 2017 mais de 350000 pacotes foram reportados como o número total de pacotes existentes no registo do npm, tornando-o o maior repositório de código de uma única linguagem na Terra, e podes estar certo de que existe um pacote para (quase!) tudo.'
+authors: flaviocopes, MylesBorins, LaRuaNa, jgb-solutions, amiller-gh, ahmadawais, nazarepiedady
+category: learn
+---
+
+<Alert>Lembra-te de que isto é uma breve introdução ao <code>NPM</code>. Para um guia mais profundo, consulte a <a href="https://docs.npmjs.com/">documentação do NPM</a>.</Alert>
+
+## Introdução ao npm
+
+O `npm` é o gestor de pacote padrão para a Node.js.
+
+Em Setembro de 2022 mais de 2.1 milhões de pacotes foram reportados como o número total listado no registo do npm, tornando-o o maior repositório de código de uma única linguagem na Terra, e podes estar certo de que existe um pacote para (quase!) tudo.
+
+Este começou como uma maneira de descarregar e gerir dependências dos pacotes da Node.js, mas tornou-se desde então uma ferramenta usada também na JavaScript do frontend.
+
+> [**Yarn**](https://yarnpkg.com/en/) e [**pnpm**](https://pnpm.io) são alternativas para interface de linha de comando do npm. Tu podes consultá-los também.
+
+## Pacotes
+
+O `npm` lida com os descarregamentos de dependências do teu projeto.
+
+### Instalando todas as dependências
+
+Se um projeto tiver um ficheiro `package.json`, com a execução de:
+
+```bash
+npm install
+```
+
+ele instalará tudo que o projeto precisa, na pasta `node_modules`, criando-a se já não existir.
+
+### Instalando um único pacote
+
+Tu podes também instalar um pacote específico executando:
+
+```bash
+npm install <package-name>
+```
+
+Além disto, desde a versão 5 do npm, este comando adiciona o `<package-name>` ao objeto _`dependencies`_ do ficheiro `package.json`. Antes da versão 5, precisavas de adicionar a opção `--save`.
```

Comment:
```suggestion
Além disto, desde a versão 5 do npm, este comando adiciona o `<package-name>` ao objeto _`dependencies`_ do ficheiro `package.json`. Antes da versão 5, precisava adicionar a opção `--save`.
```

Reply:
I do not see the necessity of changing it, and since there are no grammar or syntactical errors, it should remain as it is.